Makefile

```diff
@@ -6,6 +6,11 @@ GOLANGCI_LINT := $(BIN_DIR)/golangci-lint
 GOLINT := $(BIN_DIR)/golint
 MOCKGEN := $(BIN_DIR)/mockgen
 
+VERSION= $(shell git describe --tags)
```

Comment:
That could possibly not work - it does in our case but `--always` could be safer